packages/opencensus-web-instrumentation-zone/src/zone-types.ts

```diff
@@ -38,6 +38,13 @@ export interface OnPageInteractionData {
   rootSpan: RootSpan;
 }
 
+/** Allows monkey-patching XMLHttpRequest and to obtain the request URL. */
+export type XHRWithUrl = HTMLElement &
```

Comment:
Why does this need the `HTMLElement` type here?

Reply:
`HTMLElement` is necessary when the xhr is captured from the tasks target as the Zone monkey-patch parses xhrs as `HTMLElement & XMLHttpRequest`. 
Added this as a comment also.